### ISSUE TITLE
Nix needs more maintainers

### ISSUE THREAD

**Issue by asomers** [nix-rust/nix#1304]
You may have noticed that my responsiveness has been lagging the past few months.  COVID has given me less time to work on Nix, though TBH I've been burning out, too.  Would somebody volunteer to help me with this crate?  Old maintainers are welcome, as are outside contributors with good track records.
cc @Susurrus @posborne @kamalmarhubi

**Comment by posborne** [comment]
@asomers Thanks for all your work and I understand where you are at and can definitely relate!  I'm happy to at least make a solid attempt to step up here and help out more, but I agree that it feels right to seek out a few new, active maintainers to keep things moving and healthy.

**Comment by elmarco** [comment]
I don't feel like I am up for the task, sorry :) Any plan for release nevertheless?

**Comment by kamalmarhubi** [comment]
@asomers thanks for opening this issue! As an experiment, I will try to respond for the month of October and see how I feel. Assume a few days turnaround time.

@elmarco I just `bors r+`'d a PR to put out 0.19 to unblock @qwandor-google. Hopefully that works!

**Comment by woodruffw** [comment]
I'm happy to offer both review and maintenance time here (professionally, since I rely on this crate at work for a variety of tools). I don't have any track record *here*, but feel free to snoop my GitHub activity to vet me 🙂

**Comment by Susurrus** [comment]
I've been stepping back from my programming responsibilities as well. At this point it's probably best that I leave as a maintainer entirely as I'm really not able to spend any time on this and don't want to give the illusion of being available. I'll wait to leave officially until we make more progress here with this discussion. Has anyone posted on IRC or r/rust about this? Getting it in the weekly Rust newsletter might also be a good idea. I'm happy to do that if nobody has already done so.

**Comment by ranweiler** [comment]
I'd like to volunteer as an additional maintainer. I depend on this library for various projects, and would rather contribute to its success than have to fork it or fall back on raw `libc`!

**Comment by ranweiler** [comment]
@asomers, any thoughts on how we can help out moving forward?

**Comment by anth1y** [comment]
I'd love to jump in and help out i'm pretty new to rust but i'm definitely down to help out where I can with this crate @asomers @posborne

**Comment by nox** [comment]
@asomers I know I didn't contribute much to this crate but nonetheless I volunteer to help maintain it. I am currently at Cloudflare where we use nix quite extensively for fancy stuff such as heavy use of the recvmmsg/sendmmsg APIs. What do you think?

**Comment by krobelus** [comment]
Hi, I'd be happy to help out with maintenance as well.
Working on fish shell, we've run into several issues that we want to address eventually.
[One of them](https://github.com/fish-shell/fish-shell/issues/12458) is https://github.com/nix-rust/nix/pull/2799  -- Fix incorrect "From<Termios> for libc::termios" on Illumos -- which looks reasonable to me but has been waiting for review for a while.
I also ran into usability issues with the sigaction and Errno APIs but haven't prioritized those.
